docs/bbs/worlds.md

```diff
@@ -2,28 +2,429 @@
 
 |Id|Index|World
 |--|-----|--
-|BD|10   |Command Board
-|CD|03   |Castle of Dreams
-|DC|0C   |Disney Town
-|DI|0A   |Destiny Island
-|DP|01   |Land of Departure
 |EX|00   |Generic
-|HE|08   |Olympus Coliseum
-|JB|07   |Jungle Book (JP version)
-|JB|07   |Dark World (FM version)
-|JF|17   |???
-|KG|0D   |Keyblade Graveyard
-|LS|09   |Deep Space
-|PP|0B   |Never Land
-|RG|06   |Radiant Garden
-|SB|04   |Enchanted Domain
-|SW|02   |Dwarf Woodlands
-|VS|0F   |Mirage Arena
-|WM|11   |World Map
-|WP|12   |???
-|YT|05   |Mysterious Tower
+|DP|01   |[Land of Departure](#Land-Of-Departure)
+|SW|02   |[Dwarf Woodlands](#Dwarf-Woodlands)
+|CD|03   |[Castle of Dreams](#Castle-Of-Dreams)
+|SB|04   |[Enchanted Dominion](#Enchanted-Dominion)
+|YT|05   |[Mysterious Tower](#Mysterious-Tower)
+|RG|06   |[Radiant Garden](#Radiant-Garden)
+|JB|07   |[Jungle Book (JP version)](#FOR-EVENTS-ONLY)
+|JB|07   |[Dark World (FM version)](#FOR-EVENTS-ONLY)
+|HE|08   |[Olympus Coliseum](#Olympus-Coliseum)
+|LS|09   |[Deep Space](#Deep-Space)
+|DI|0A   |[Destiny Island](#Destiny-Islands)
+|PP|0B   |[Neverland](#Neverland)
+|DC|0C   |[Disney Town](#Disney-Town)
+|KG|0D   |[Keyblade Graveyard](#Keyblade-Graveyard)
 |14|0E   |unused
+|VS|0F   |[Mirage Arena](#Mirage-Arena)
+|BD|10   |[Command Board](#Command-Board)
+|WM|11   |[World Map](#World-Map)
+|WP|12   |[Winnie the Pooh](#Winnie-the-Pooh)
 |19|13   |unused
 |20|14   |unused
 |21|15   |unused
-|22|16   |unused
\ No newline at end of file
+|22|16   |unused
+|JF|17   |[Jump Festa](#Jump-Festa)
+
+# Map List
```

Comment:
You might want to change from `#` to `##` as `#` should only be used at the very top as a title (markdown language specifications)

Reply:
Alright!